### ISSUE TITLE
Tag packages with languages used

### ISSUE THREAD

**Issue by jonasfj** [dart-lang/pub-dartlang-dart#2154]
This is not fully though through. But github has a feature where it shows which language a repositories is written in..

It might be nice to have the analysis of a package. Just look for `*.dart|py|java|kt|kts|m|mm|M|h|c|cc|cpp|swift|js` and show percentages maybe?

This would be useful insights for flutter packages. See https://github.com/dart-lang/pub-dartlang-dart/issues/187#issuecomment-477161797. I'm not sure searching is useful, but maybe...

**Comment by jaumard** [comment]
Thanks for the issue @jonasfj ! Searching no, but filtering can be really useful :) 
For example if I'm search GPS plugin, I'll type geolocation on the searchbar but having a filter to only show java/objC entries will ensure me to have no problem with Kotlin and or Swift setup/number version mismatch, for me it's a huge advantage even more for a professional / company usage

**Comment by isoos** [comment]
This could be useful, but only after we can filter out the Flutter-generated template files, that may not be in use, but left around in the package.

**Comment by jonasfj** [comment]
> after we can filter out the Flutter-generated template files, that may not be in use, but left around in the package.

Or we should punish that we a lower package score?

**Comment by sigurdm** [comment]
Why is this interesting for the package consumer?
